luigi/contrib/k8s_job.py

```diff
@@ -186,8 +216,14 @@ def run(self):
                 }
             }
         }
+        job_json['metadata']['labels'].update(self.labels)
         job = Job(self.__kube_api, job_json)
         job.create()
         # Track the Job (wait while active)
         self.__logger.info("Start tracking Kubernetes Job: " + self.uu_name)
         self.__track_job()
+
+    def output(self):
```

Comment:
I'd like this to be optional as well. Please remove this the `output` method override. Let's say I write into a DB, and I redefine the `complete` method to check that there is a result record in the DB (instead of letting Luigi to check an output file). Then, I should be allowed to not define the output.